### ISSUE TITLE
Keyboard focus indicator is not visible on tab controls (Paris, New York..) in windows.

### ISSUE THREAD

**Issue by msftedad** [telerik/kendo-ui-core#8414]
**Test Environment:**
OS: Windows 11 version 25H2(OS Build 26220.7271)
Browser: Edge (Version 143.0.3650.66)
URL: https://dojo.telerik.com/vBjelqpZ

**User Impact:**
Keyboard-only users cannot identify which tab control (e.g., Paris, New York) currently has focus because the visual indicator is missing. This makes navigation confusing and increases the risk of selecting the wrong tab.

**Repro Steps:**

1. Open the URL: https://dojo.telerik.com/vBjelqpZ in edge browser in windows.
2. Editor screen will appear.
3. Press tab key to navigate to the 'Run' button and activate it.
4. Press tab key to navigate to the tab control (e.g., Paris, New York).
5. Verify whether keyboard focus indicator is visible on tab controls (Paris, New York..) in windows or not.

**Actual Result:**
Keyboard focus indicator is not visible on tab controls (Paris, New York..) in windows.

**Refer Attachment:**

1. Keyboard focus indicator is not visible on tab controls (Paris, New York..) in windows.png
2. Keyboard focus indicator is not visible on tab controls (Paris, New York..) in windows.mp4

<img width="970" height="595" alt="Image" src="https://github.com/user-attachments/assets/8584b7b6-4f51-4d61-a6b4-e278fb3d145f" />

https://github.com/user-attachments/assets/ff2bfac7-cda6-4ed1-b2ec-d5d61051f1e6

**Expected Result:**
Keyboard focus indicator should be visible on tab controls (Paris, New York..) in windows.

**Comment by Dimitar-Goshev** [comment]
@msftedad Thank you for your detailed report.
This is a [known issue](https://github.com/telerik/kendo-ui-core/issues/7399) and it was fixed in release 2024 Q2. The fix is available since version 2024.1.319 and kendo themes 7.2.1 - [Dojo](https://dojo.telerik.com/eHlQnMlI)

**Comment by msftedad** [comment]
Hi @Dimitar-Goshev ,Issue is still repro'ing in [Dojo](https://dojo.telerik.com/eHlQnMlI)

**Comment by msftedad** [comment]
Hi @Dimitar-Goshev, Any update here?
